templates/next/pages/foos/[id]/index.tsx

```diff
@@ -27,21 +30,22 @@ const Page: NextComponentType<NextPageContext, Props, Props> = (props) => {
       </div>
       <Show {{{lc}}}={ {{{lc}}} } text={ props.text } />
```

Comment:
There are no props any more and text is not exposed as well from them.

Reply:
This issue is corrected on [319](https://github.com/api-platform/client-generator/pull/319)